packages/fxa-settings/src/components/Profile/mocks.tsx

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Account } from '../../models';
+import { mockEmail } from '../../models/mocks';
+
+export const MOCK_PROFILE_EMPTY = {
+  displayName: null,
+  avatar: { id: null, url: null },
+  primaryEmail: mockEmail(),
+  emails: [mockEmail()],
+} as unknown as Account;
```

Comment:
In my next PR I might consider separating our `AccountData` types a bit better and/or file follow up ticket(s)... I think there's some improvements to be had here.

Reply:
Excited to hear more!